src/pandas_profiling/visualisation/plot.py

```diff
@@ -3,18 +3,17 @@
 from typing import Union, Optional
 
 import matplotlib
+import matplotlib.pyplot as plt
```

Comment:
I see the logic in the other import changes, but why is `matplotlib.colors` not grouped with the other matplotlib imports here?

In the future, it is better to put these code style changes (organizing the imports) into a separate PR (for easier review).

Reply:
In fact, I don't know. It seems that all import statements are sorted alphabetically, and this statement start with `from`. My IDE did this job automatically and I committed it because I think it looks good.

Thanks for your advice, I will put code style changes into a separate PR in the future.